lmfit/minimizer.py

```diff
@@ -591,16 +589,16 @@ def __residual(self, fvars, apply_bounds_transformation=True):
                                  *self.userargs, **self.userkws)
             self._abort = self._abort or abort
 
-        if self._abort:
-            self.result.residual = out
-            self.result.aborted = True
-            self.result.message = "Fit aborted by user callback. Could not estimate error-bars."
-            self.result.success = False
-            raise AbortFitException("fit aborted by user.")
-        else:
+        if not self._abort:
             return _nan_policy(np.asarray(out).ravel(),
                                nan_policy=self.nan_policy)
 
+        self.result.residual = out
+        self.result.aborted = True
+        self.result.message = "Fit aborted by user callback. Could not estimate error-bars."
+        self.result.success = False
+        raise AbortFitException("fit aborted by user.")
+
```

Comment:
please revert this block to the original; there is no benefit of doing this with respect to speed / readability